utils/pdfgeneration.js

```diff
@@ -160,4 +160,4 @@ const generatePdfForSubscriber = async (
   }, Constants.PDF_DELETE_DELAY * 1000);
 };
 
-export default generatePdfForSubscriber;
+export {generatePdfForSubscriber};
```

Comment:
Destructured import/export to enable Sinon stubbing

Reply:
spaces before/after brackets =P